sanic/request.py

```diff
@@ -253,6 +258,33 @@ def _get_address(self):
         self._ip = self._socket[0]
         self._port = self._socket[1]
 
+    @property
+    def server_name(self):
+        """
+        server name.
+        The difference between `Request.server_name` and `Request.host`
```

Comment:
Can you please ensure you use the right docstring formatting so that the API documentation is rendered properly?

Reply:
updated